### PR TITLE
Update section about expanding Boskos pool

### PR DESCRIPTION
**Pull request by adrcunha**

### CODE REVIEW

File: ci/prow_setup.md [hunk 1]
```diff
@@ -38,12 +38,12 @@
 1. Run `make update-boskos-config` to update the Boskos config.
 
 1. Increase the compute CPU quota for the project to 200. Go to
```
Comment (chaodaiG) [inline ci/prow_setup.md]: it's not related to this PR, but just noticed this step should swap with the step above so there is no surprise of the new project picks up performance test before quota increase kicks in

Comment (adrcunha) [reply ci/prow_setup.md]: Thanks. Done.

**Pull request merged**